music21/stream/base.py

```diff
@@ -7287,6 +7288,21 @@ def stripTies(
         posConnected = []  # temporary storage for index of tied notes
         posDelete = []  # store deletions to be processed later
 
+        def pitchesEqual(p1, p2):
```

Comment:
since this is redundant code from the previous PR, we should factor it out -- I'm okay with having it as a method on Pitch that is used in eq... wait -- I'm going to do that right now...